src/content/docs/codestream/codestream-ui-overview/permalinks.mdx

```diff
@@ -3,41 +3,43 @@ title: CodeStream permalinks
 metaDescription: "CodeStream's permalinks are the mechanism used for sharing to specific comments, issues, and feedback requests to your issue-tracking service."
 ---
 
-## Permalinks for Sharing Anywhere
+New Relic CodeStream's permalinks can be shared anywhere you'd like to share a link.
```

Comment:
This sentance is a bit messy. No real ideas on how to improve, so feel free to ignore.

Reply:
A fair comment.